### ISSUE TITLE
golang apimanagement Schema is missing

### ISSUE THREAD

**Issue by asalkeld** [pulumi/pulumi-azure-native#1513]
## Hello!

- Vote on this issue by adding a 👍 reaction
- To contribute a fix for this issue, leave a comment (and link to your pull request, if you've opened one already)

## Issue details

Here there is a reference to Schema{}
https://github.com/pulumi/pulumi-azure-native/blob/master/sdk/go/azure/apimanagement/init.go#L108-L109
But it does not exist.

### Steps to reproduce

1. run a go program with apimanagement

Expected: no build issues
Actual: 

```
 # github.com/pulumi/pulumi-azure-native/sdk/go/azure/apimanagement
../../../../pkg/mod/github.com/pulumi/pulumi-azure-native/sdk@v1.58.0/go/azure/apimanagement/init.go:109:8: undefined: Schema
```

**Comment by viveklak** [comment]
Weird - running `make go_build` locally with v1.58.0 results in:
```
make build_go
# Only building the top level packages and building 1 package at a time to avoid OOMing
cd sdk/ && \
	GOGC=50 go list github.com/pulumi/pulumi-azure-native/sdk/go/azure/... | grep -v "latest\|\/v.*"| xargs -L 1 go build
# github.com/pulumi/pulumi-azure-native/sdk/go/azure/apimanagement
go/azure/apimanagement/init.go:109:8: undefined: Schema
# github.com/pulumi/pulumi-azure-native/sdk/go/azure/securityinsights
go/azure/securityinsights/init.go:67:8: undefined: Metadata
make: *** [build_go] Error 1
```

However, the tag build step didn't complain at all: https://github.com/pulumi/pulumi-azure-native/runs/5191901260?check_suite_focus=true#step:15:27

@stack72 any idea why this is inconsistent?

**Comment by mikhailshilkov** [comment]
@danielrbradley Could you summarize the issue with codegen? What's the difference between the default version and the explicit one?

**Comment by asalkeld** [comment]
Thank you @danielrbradley , that's awesome!

**Comment by danielrbradley** [comment]
This is now available in v1.59.0
